### ISSUE TITLE
Please turn MAX_SUPER_INDEX_ENTRIES into an optional parameter

### ISSUE THREAD

**Issue by TomArrow** [baSSiLL/SharpAvi#45]
I just had the creation of an AVI file fail after 2 days and about 1 TB of data written. 

```
System.InvalidOperationException
  HResult=0x80131509
  Message=Cannot write more frames to this stream.
  Source=SharpAvi.NetStandard
  StackTrace:
   at SharpAvi.Output.AviWriter.WriteStreamFrame(AviStreamBase stream, Boolean isKeyFrame, Byte[] frameData, Int32 startIndex, Int32 count)
   at SharpAvi.Output.AviWriter.SharpAvi.Output.IAviStreamWriteHandler.WriteVideoFrame(AviVideoStream stream, Boolean isKeyFrame, Byte[] frameData, Int32 startIndex, Int32 count)
   at SharpAvi.Output.AviVideoStream.WriteFrame(Boolean isKeyFrame, Byte[] frameData, Int32 startIndex, Int32 count)
   at SharpAvi.Output.VideoStreamWrapperBase.WriteFrame(Boolean isKeyFrame, Byte[] frameData, Int32 startIndex, Int32 length)
   at SharpAvi.Output.AsyncVideoStreamWrapper.<>n__0(Boolean isKeyFrame, Byte[] frameData, Int32 startIndex, Int32 length)
   at SharpAvi.Output.AsyncVideoStreamWrapper.<>c__DisplayClass2_0.<WriteFrame>b__0()
   at SharpAvi.Output.SequentialInvoker.Invoke(Action action)
   at SharpAvi.Output.AsyncVideoStreamWrapper.WriteFrame(Boolean isKeyFrame, Byte[] frameData, Int32 startIndex, Int32 length)

  This exception was originally thrown at this call stack:
    [External Code]
```

I believe this is because I exceeded the maximum amount of super index entries:
```cs
if (si.SuperIndex.Count == MAX_SUPER_INDEX_ENTRIES)
{
    throw new InvalidOperationException("Cannot write more frames to this stream.");
}
```

From what quickly googled, this appears to be some kind of safety measure for compatibility? Can you please make it so that one can optionally set this to something higher and maybe add a kind of "Only use if you know what you are doing" warning to it? 

I'll try to do this on a fork and if I succeed, I'll maybe send a PR.

**Comment by baSSiLL** [comment]
Published in v2.2.0.

**Comment by TomArrow** [comment]
Cool, thanks!
